.travis.yml

```diff
@@ -0,0 +1,55 @@
+language: python
+python:
+  - "2.7"
+sudo: required
+services:
+  - docker
+env:
+  global:
+    - PA_TEST_ONLINE_INSTALLER=true
+    - PYTHONPATH=$(pwd)
+  matrix:
+    - OTHER_TESTS=true
+    - PRODUCT_TEST_GROUP_SA_BARE="tests/product/test_installer.py tests/product/standalone/test_installation.py"
```

Comment:
Yea, maybe you can rename the groups to `STANDALONE_BARE_TESTS` and `YARN_SLIDER_TESTS`. Including the `PRODUCT_TEST_GROUP_*` prefix is a little too verbose and redundant.